app/concerns/koudoku/subscription.rb

```diff
@@ -221,15 +221,15 @@ def subscription_owner_email
   end
 
   def changing_plans?
-    plan_id_changed?
+    will_save_change_to_plan_id?
```

Comment:
Should this be `saved_change_to_plan_id?`?

Reply:
This is a tricky one because this is called in a `before_save` callback so the old `_changed?` would return a true here if `plan_id` has changed before save instead `saved_change_to_plan_id?` will be false. The correct one is `will_save_change_to_plan_id?` - hope it makes sense.
I have struggled a lot in the mothership to fix all of those because you need to double check if before/after save